versioned_docs/version-8.2/reference/dependencies.md

```diff
@@ -1732,237 +1732,203 @@ Desktop Modeler is a desktop modeling application that builds upon a number of t
 
 ### Web Modeler Dependencies (webapp)
```

Comment:
@JacobiCamunda I think you ran this command on the wrong branch. This list contains dependency versions that don't reflect the ones used in `8.2.8`.

Reply:
Mhm, I will take a look and do this again